src/tailwindcss.worker.ts

```diff
@@ -111,10 +115,15 @@ export function initialize(tailwindWorkerOptions?: TailwindWorkerOptions): void
       tailwindWorkerOptions?.prepareTailwindConfig?.(options.tailwindConfig) ??
       options.tailwindConfig ??
       ({} as TailwindConfig);
+    if (typeof preparedTailwindConfig !== 'object') {
+      throw new TypeError(
+        `Expected tailwindConfig to resolve to an object, but got: ${JSON.stringify(
+          preparedTailwindConfig,
+        )}`,
+      );
+    }
 
-    const config = resolveConfig(preparedTailwindConfig);
-
-    const state = stateFromConfig(config);
+    const statePromise = stateFromConfig(preparedTailwindConfig);
```

Comment:
idk how if this is a good pattern to encode the variable type in the name `*Promise` - what do you say?
(i get the point, as not everyone looks at this code in an IDE)